ls1intum/Artemis
# show grading instructions in tutor dashboard and exercise view for all exercise types

### Checklist

### Motivation and Context


Resolve #1090 
### Description

Grading instructions should be displayed consistently throughout all exercise types in the exercise dashboard and exercise view.
### Steps for Testing


1. Navigate to any exercise type (apart from automatic programming exercise)
2. Click on view
-Problem statement
-Example Solution
-Grading Criteria
should be displayed in this order
3. Click on Tutor Course Dashboard
Click on Exercise Dashboard. Exercise instructions should be displayed there.

### Screenshots
